File: 2824-count-pairs-whose-sum-is-less-than-target/2824-count-pairs-whose-sum-is-less-than-target.c
int countPairs(int* nums, int numsSize, int target){
    int count=0, j, i;
    for(i=0; i<numsSize; i++)
{
    for(j=0; j<numsSize; j++)
    {
    if(nums[i]+nums[j]<target && i<j)
           {
               count++;
           }
    }
}
    return count;

}

File: 2824-count-pairs-whose-sum-is-less-than-target/2824-count-pairs-whose-sum-is-less-than-target.java
class Solution {
    public int countPairs(List<Integer> nums, int target) {
            int count=0, j, i;
    for(i=0; i<nums.size(); i++)
    {
        for(j=0; j<nums.size(); j++)
        {
            if(nums.get(i)+nums.get(j)<target && i<j)
               count++;
        }
    }
    return count;

    }
}